phylogenetic/rules/merge_inputs.smk

```diff
@@ -0,0 +1,184 @@
+"""
+This part of the workflow merges inputs based on what is defined in the config.
+
+OUTPUTS:
+
+    metadata  = "results/metadata.tsv"
+    sequences = "results/sequences.fasta"
+
+The config dict is expected to have a top-level `inputs` list that defines the
+separate inputs' name, metadata, and sequences. Optionally, the config can have
+a top-level `additional-inputs` list that is used to define additional data that
+are combined with the default inputs:
+
+```yaml
+inputs:
+    - name: default
+      metadata: <path-or-url>
+      sequences: <path-or-url>
+
+additional_inputs:
+    - name: private
+      metadata: <path-or-url>
+      sequences: <path-or-url>
+```
+
+Supports any of the compression formats that are supported by `augur read-file`,
+see <https://docs.nextstrain.org/projects/augur/page/usage/cli/read-file.html>
+
+NOTE: The included rules are written for single build workflows such
+as zika that do not use wildcards. You will need to edit the rules to support wildcards
```

Comment:
Isn't the restriction based on input data, not how many builds are in the workflow? WNV is multi-build but it works there because all the builds use the same input data. Suggested rewording:

> NOTE: The included rules are written for workflows that use a single input without wildcards. You will need to edit the rules to support wildcards …